### ISSUE TITLE
Question 4a confusing

### ISSUE THREAD

**Issue by netimmerman** [NUCS349/hw4-gradient-descent#1]
The question states:
"Set the step size, or _learning_rate, to 1e-5_ and the **maximum number of iterations to 2000**. Report your values for how many digits from each class you used, what your batch size was, **and how many iterations you used**. For each regularizer, run the algorithm once for each of the following values for lambda: _[1e-3, 1e-2, 1e-1, 1, 10, 100]_."

A couple pieces of information seem conflicting: set the learning_rate to 1e-5, which isn't a value included in the subsequent list. It also says to set the max_iters to 2000, then in the next sentence asks you to report how many iterations you used despite it telling you how many to use.

**Comment by ethman** [comment]
The learning rate (or step size) and lambda (or regularization parameter) are different things. [See slide 9 for the definition of the learning rate/step size, and slide 28 for definition of the regularization parameter/lambda.](https://nucs349.github.io/lectures/cs349_gradient_descent.pdf) Also, the maximum number of iterations is not the number of iterations that your algorithm may take because it should stop upon converging (see the documentation for convergence criteria).
